packages/assets-controllers/src/Standards/NftStandards/ERC1155/ERC1155Standard.ts

```diff
@@ -192,6 +256,8 @@ export class ERC1155Standard {
       standard: ERC1155,
```

Comment:
The above TODO might be worth doing as well:
`// TODO consider querying to the metadata to get name.`

In our use case we're doing a collection level approval, and don't have a specific token id to get metadata from.  But if we *did*, we may want the token's metadata name to take precedence over the contract name (if one exists)